tests/test_fast_setup/test_fast_setup.py

```diff
@@ -8,15 +9,19 @@
 
 log = get_logger(__name__)
 
-mrsal = Mrsal(
-    # host=test_config.HOST,
-    host=config.RABBIT_DOMAIN,
-    port=config.RABBITMQ_PORT_TLS,
-    ssl=True,
-    credentials=config.RABBITMQ_CREDENTIALS,
-    virtual_host=config.V_HOST
-)
+# mrsal = Mrsal(
+#     # host=test_config.HOST,
+#     host=config.RABBIT_DOMAIN,
+#     port=config.RABBITMQ_PORT_TLS,
+#     ssl=True,
```

Comment:
I need help here. I couldn't connect to **RabbitMQ** server using TLS.

Reply:
what is the error?

Reply:
Don't remember precisely. But it something like **authentication refused**.
Will double check and come back to you with the next commit ;)

Reply:
What is the config? Are you using the correct domain?